readthedocs/search/indexes.py

```diff
@@ -129,17 +127,14 @@ def bulk_index(self, data, index=None, chunk_size=500, parent=None,
             doc = {
                 '_index': index,
                 '_type': self._type,
-                '_id': source['id'],
                 '_source': source,
             }
-            if parent:
-                doc['_parent'] = parent
```

Comment:
How the parent child relationship will be made?

Reply:
This was breaking in ES5 when I tested it. I think it will require a bit more thought when we move towards ES6, which is a fundamental change in how ES works, so it will require refactoring and thinking about architecture more.